frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.test.ts

```diff
@@ -145,8 +145,8 @@ describe('sessionRecordingDataLogic', () => {
                 .toDispatchActions(['loadRecordingMeta', 'loadRecordingMetaSuccess', 'loadEvents'])
                 .toMatchValues({
                     eventsApiParams: {
-                        after: '2021-12-09T19:35:59+00:00',
-                        before: '2021-12-09T20:23:24+00:00',
+                        after: '2021-12-09T19:35:59Z',
+                        before: '2021-12-09T20:23:24Z',
```

Comment:
@mariusandra I had to make this change here. I spent a good hour looking into why but I couldn't figure it out. It seems like a pretty pure code path calling `dayjs.utc(TIME).format()` where `TIME` hasn't changed in this PR (I checked).

Feels like somewhere dayjs is being mutated in a way that this PR has modified but I can't figure out why. Perhaps you have some ideas?